server/rest/src/main/java/org/infinispan/rest/resources/StaticContentResource.java

```diff
@@ -115,4 +113,8 @@ private File resolve(String resource) {
             .entity(file);
       return completedFuture(responseBuilder.build());
    }
+
+   private boolean isInfinispanConsolePath(String uri) {
+      return uri.equals(rootUri) || (uri.contains(rootUri) && !uri.contains("."));
```

Comment:
@karesti I'm still not sure what the "." check is for, or why we don't use just `uri.startsWith(rootUri)`

Reply:
when we ask for the 'console' with /console/ or /console, the server gets the index file which brings the react app. in the react app we resolve in the front for different url paths (console/create/cache or whatever url). So any path that belongs to the console, needs to get the index static file.
However, if we ask for a static resource (css, image etc...) then this file needs to be loaded. And these are also under 'console' path.
The simple and straightforward quick way I found to distinguish now between a path and a file request is having the '.' in the url. this can be polished of course

Reply:
If the resource doesn't end with a `/` then it's a file resource, otherwise it's a path resource. There shouldn't be any of those: the actual console application should default to using `/` for those resources to avoid ambiguity (web servers usually redirect to `uri/`).